src/task2/middlewares/errorHandlerMiddleWare.js

```diff
@@ -0,0 +1 @@
+export default (err, req, res, next) => err ? res.status(500).send(err.stack) : next();
```

Comment:
Consider either storing the http status codes of your project in the core directory or using http-status-codes dependency.